ci/nightly/build-tiledbvcf-py.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -ex
+
+# Build tiledbvcf-py assuming source code directory is ./TileDB-VCF/apis/python
+
+export LD_LIBRARY_PATH=$GITHUB_WORKSPACE/install/lib:${LD_LIBRARY_PATH-}
```

Comment:
Do you know why this is necessary? cc @awenocur. I see there are other usages in the CI setup. This may mean that we are not configuring RPATH correctly.